app/views/finance/participants/_npq_application.html.erb

```diff
@@ -68,14 +68,4 @@
     <% row.with_value { application.updated_at.to_fs(:govuk) } %>
     <% row.with_action(text: :none) %>
   <% end %>
-
-  <% summary_list.with_row do |row| %>
```

Comment:
Shouldn't we delete these files instead?
I think we render the partial on `app/views/finance/participants/show.html.erb`

Reply:
Well I thought of doing that but was trying to focus on removing the things related to the API endpoints only in this card.